test/crypto.test.ts

```diff
@@ -0,0 +1,57 @@
+const passphrase = 'pwd1234';
+
+describe('publicEncrypt() and privateDecrypt()', () => {
+    it('should decrypt encrypted random strings', () => {
+        for (let i = 0; i < 100; i++) {
+            const randomString = utils.cryptoRandomObjectId(10);
+            const { encryptedPassword, encryptedValue } = utils.publicEncrypt(publicKey, randomString);
+            const decryptedValue = utils.privateDecrypt(privateKey, passphrase, encryptedPassword, encryptedValue);
+            expect(randomString).toEqual(decryptedValue);
+        }
+    });
+
+    it('should decrypt encrypted random strings with special characters', () => {
+        const randomString = utils.cryptoRandomObjectId(10);
+        // eslint-disable-next-line max-len
+        for (const char of ['👍', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '-', '_', '=', '+', '[', ']', '{', '}', '|', ';', ':', '"', "'", ',', '.', '<', '>', '?', '/', '~', '`']) {
+            const stringWithSpecialChar = `${char}${randomString}${char}${randomString}${char}`;
+            const { encryptedPassword, encryptedValue } = utils.publicEncrypt(publicKey, stringWithSpecialChar);
+            const decryptedValue = utils.privateDecrypt(privateKey, passphrase, encryptedPassword, encryptedValue);
+            expect(stringWithSpecialChar).toEqual(decryptedValue);
+        }
+    });
+
+    it('throws if encrypted password is not valid', () => {
+        const randomString = utils.cryptoRandomObjectId(10);
+        const { encryptedPassword, encryptedValue } = utils.publicEncrypt(publicKey, randomString);
+        expect(() => utils.privateDecrypt(privateKey, passphrase, encryptedPassword.slice(2), encryptedValue)).toThrow();
+        expect(() => utils.privateDecrypt(privateKey, passphrase, `bla${encryptedPassword}`, encryptedValue)).toThrow();
+    });
+
+    it('throws if encrypted value is not valid', () => {
+        const randomString = utils.cryptoRandomObjectId(10);
+        const { encryptedPassword, encryptedValue } = utils.publicEncrypt(publicKey, randomString);
+        expect(() => utils.privateDecrypt(privateKey, passphrase, encryptedPassword, encryptedValue.slice(2))).toThrow();
+        expect(() => utils.privateDecrypt(privateKey, passphrase, encryptedPassword, `bla${encryptedValue}`)).toThrow();
+    });
+
+    it('should return different cipher for the same string', () => {
+        const randomString = utils.cryptoRandomObjectId(100);
+        const uniqueCiphers = new Set();
+        const uniqueCiphers16Bytes = new Set();
+        for (let i = 0; i < 100; i++) {
+            const { encryptedPassword, encryptedValue } = utils.publicEncrypt(publicKey, randomString);
+            const theFirst16Bytes = Buffer.from(encryptedValue, 'base64').toString('utf-8').slice(0, 16);
+            expect(uniqueCiphers.has(encryptedPassword)).toBe(false);
+            expect(uniqueCiphers16Bytes.has(theFirst16Bytes)).toBe(false);
+            uniqueCiphers.add(encryptedValue);
```

Comment:
Are you checking the uniqueness of `encryptedValue`, or `encryptedPassword`?

Reply:
encryptedValue, there is a typo, will fix it.